store/event-store/src/main/java/org/calrissian/accumulorecipes/eventstore/impl/AccumuloEventStore.java

```diff
@@ -301,4 +301,45 @@ public void flush() throws Exception {
     public CloseableIterable<String> getTypes(String prefix, Auths auths) {
         return helper.getKeyValueIndex().getTypes(prefix, auths);
     }
+
+    public static class Builder {
```

Comment:
I actually like the builder idea, but it should probably be brought in as part of another MR.  It should also include the identical EntityStore.